config/settings.yml

```diff
@@ -61,4 +61,6 @@ use_custom_pdf_viewer: true
 use_custom_3d_viewer: true
 collections_to_show_attribution:
   - dc876nn4469
-analytics_id:
+# Google Analytics will report in debug mode by default
+# in shared_configs for embed-prod, we set this value to 0  
+analytics_debug: 1
```

Comment:
Is there a reason this isn't `true`?  Why do we configure this as a integer?

Reply:
Yes when we implemented this same debug flag in VT we ran into an issue and solved it using integer, so I went with the same thing here.

Reply:
What issue?  It seems unnecessarily complex that we have to use javascript to convert a string to a boolean to handle this, when the config gem itself can do this for us.

Reply:
@jcoyne I don't know a good way to store this in a setting/config file for use in JS. Sure we can store it in in the config file as a Bool but in JS it's going to load as a string. It seems more complex to do logic like if "true" then ... -- than to just store a 1 or 0. Maybe I'm missing something obvious tho.

Reply:
Boolean("false") in JS still evaluates to true -- 0/1 seemed safer

Reply:
Boolean("false") in js still evaluates to true -- 1/0 seemed safer

Reply:
Why do you need to wrap it in Boolean? just render the boolean in the JS.

Reply:
It looks like I made an error here, but I still think we need to use Boolean.

`typeof('<%= Settings.analytics_debug %>'))` is string whether the value is "false", false, or 0.
`typeof(Number('<%= Settings.analytics_debug %>')))` is number when the value is 0.
Boolean("false") === true
Boolean(0) === false

So -- I think we need `Boolean(Number('<%= Settings.analytics_debug %>')))` to actually get a false into JS.
I agree this is complex. I don't know a better approach if we want to set the debug value out of shared_configs/settings. Maybe setting another way is better.

Reply:
@jcoyne you were right -- I was forcing it to a string by mistake. Thank you for catching that. Pushed a fix. The values in settings can be true or false, not integers.